api/src/api/v1/community_businesses/visitors/__tests__/get.test.integration.ts

```diff
@@ -8,7 +8,7 @@ import { ExternalCredentials, name as ExtName } from '../../../../../auth/strate
 import { injectCfg } from '../../../../../../tests/utils/inject';
 
 
-describe('API /community-businesses/{id}/visitors', () => {
+describe('API', () => {
```

Comment:
Seems like `GET /community-businesses/{id}/visitors` would be fine here?

Reply:
This file tests both of these routes:

<img width="523" alt="Screen Shot 2019-08-20 at 17 05 37" src="https://user-images.githubusercontent.com/14337958/63364077-c9246280-c36c-11e9-81c4-4c87bdab3ba7.png">